src/components/screens/monitor/delegates/withForgingStatus.js

```diff
@@ -79,6 +79,7 @@ const withForgingStatus = delegatesKey => (ChildComponent) => {
             nextHeight: height + i + 1,
           },
         }), {}),
+        nextForgersList: nextForgers.slice(0, 10),
```

Comment:
🐛 Storing `nextForgersList` next to `nextForgers` means that `nextForgersList` is not being updated on new block, which is happening for `nextForgers` in `componentDidUpdate`